apps/common/models/i-casework-documents.js

```diff
@@ -7,7 +7,7 @@ const config = require('../../../config');
 
 module.exports = class DocumentModel extends Model {
   url() {
-    return config.icasework.url + config.icasework.uploadpath;
+    return `${config.icasework.url}${config.icasework.uploadpath}?db=flcms`;
```

Comment:
```suggestion
return `${config.icasework.url}${config.icasework.uploadpath}?db=${encodeURIComponent(config.icasework.dbName)}`;
```
I used encodeURIComponent to safely encode the database name in the URL.

I would probably put the db name in config

Reply:
Thanks Sulthan, I have implemented this now